presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -127,6 +127,7 @@
     private boolean rcfileWriterValidate;
 
     private HiveMetastoreAuthenticationType hiveMetastoreAuthenticationType = HiveMetastoreAuthenticationType.NONE;
+    private boolean hiveMetastoreImpersonationEnabled;
```

Comment:
Move this config to `ThriftHiveMetastoreConfig` since it's specific to the Thrift metastore. I'm actually working on splitting up `HiveConfig` into separate configs for different modules.